src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -154,12 +154,12 @@ private int AnalyzeTemplate(FileInfo templateFilePath, FileInfo parametersFilePa
 
                 writer.WriteResults(evaluations, (FileInfoBase)templateFilePath, (FileInfoBase)parametersFilePath);
 
-                return 1;
+                return 0;
             }
             catch (Exception exp)
             {
                 Console.WriteLine(GetExceptionMessage(exp));
-                return -1;
+                return 1;
```

Comment:
@anaismiller This needs to be fixed. The returns here are for counting successes/failures when doing a directory scan. The CLI will not output correct counts now.